VizbeeSignInInfo.js

```diff
@@ -0,0 +1,11 @@
+export default class VizbeeSignInInfo {
+
+    constructor() {
+        this.userId = undefined;
+        this.userLogin = undefined;
+        this.userFullName = undefined;
+        this.accessToken = undefined;
+        this.refreshToken = undefined;
```

Comment:
If any of these values remain undefined, what would be the translated value at the native layer?

Reply:
Above undefined properties are not sent to the native layer